### ISSUE TITLE
DB migration running before DB init

### ISSUE THREAD

**Issue by vikassinha2019** [Netflix/dispatch#120]
I am running the installation in a fresh Ubuntu VM with docker support. After running the ./install.sh for the first time, it is failing with following error:

`Successfully built 2d9cad6cd203
Successfully tagged dispatch-local:latest

Docker images built.
Creating network "dispatch_default" with the default driver
Creating dispatch_postgres_1 ... done

Setting up database...
Starting dispatch_postgres_1 ... done
Usage: dispatch [OPTIONS] COMMAND [ARGS]...

  Command-line interface to Dispatch.

Options:
  --version  Show the version and exit.
  --help     Show this message and exit.

Commands:
  contact    All commands for contact manipulation.
  database   Container for all dispatch database commands.
  incident   All commands for incident manipulation.
  plugins    All commands for plugin manipulation.
  scheduler  Container for all dispatch scheduler commands.
  server     Container for all dispatch server commands.
  term       All commands for term manipulation.
Starting dispatch_postgres_1 ... done
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade  -> e75e103693f2, Initial migration
INFO  [alembic.runtime.migration] Running upgrade e75e103693f2 -> d0501fc6be89, empty message
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1247, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
    cursor.execute(statement, parameters)
psycopg2.errors.UndefinedTable: relation "application" does not exist

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/bin/dispatch", line 8, in <module>
    sys.exit(entrypoint())
  File "/usr/local/lib/python3.8/site-packages/dispatch/cli.py", line 811, in entrypoint
    dispatch_cli()
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/dispatch/cli.py", line 592, in upgrade_database
    alembic_command.upgrade(alembic_cfg, revision, sql=sql, tag=tag)
  File "/usr/local/lib/python3.8/site-packages/alembic/command.py", line 298, in upgrade
    script.run_env()
  File "/usr/local/lib/python3.8/site-packages/alembic/script/base.py", line 489, in run_env
    util.load_python_file(self.dir, "env.py")
  File "/usr/local/lib/python3.8/site-packages/alembic/util/pyfiles.py", line 98, in load_python_file
    module = load_module_py(module_id, path)
  File "/usr/local/lib/python3.8/site-packages/alembic/util/compat.py", line 184, in load_module_py
    spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/local/lib/python3.8/site-packages/dispatch/alembic/env.py", line 84, in <module>
    run_migrations_online()
  File "/usr/local/lib/python3.8/site-packages/dispatch/alembic/env.py", line 78, in run_migrations_online
    context.run_migrations()
  File "<string>", line 8, in run_migrations
  File "/usr/local/lib/python3.8/site-packages/alembic/runtime/environment.py", line 846, in run_migrations
    self.get_context().run_migrations(**kw)
  File "/usr/local/lib/python3.8/site-packages/alembic/runtime/migration.py", line 518, in run_migrations
    step.migration_fn(**kw)
  File "/usr/local/lib/python3.8/site-packages/dispatch/alembic/versions/d0501fc6be89_.py", line 21, in upgrade
    op.add_column("application", sa.Column("created_at", sa.DateTime(), nullable=True))
  File "<string>", line 8, in add_column
  File "<string>", line 3, in add_column
  File "/usr/local/lib/python3.8/site-packages/alembic/operations/ops.py", line 1927, in add_column
    return operations.invoke(op)
  File "/usr/local/lib/python3.8/site-packages/alembic/operations/base.py", line 374, in invoke
    return fn(self, operation)
  File "/usr/local/lib/python3.8/site-packages/alembic/operations/toimpl.py", line 132, in add_column
    operations.impl.add_column(table_name, column, schema=schema, **kw)
  File "/usr/local/lib/python3.8/site-packages/alembic/ddl/impl.py", line 237, in add_column
    self._exec(base.AddColumn(table_name, column, schema=schema))
  File "/usr/local/lib/python3.8/site-packages/alembic/ddl/impl.py", line 140, in _exec
    return conn.execute(construct, *multiparams, **params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 984, in execute
    return meth(self, multiparams, params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/sql/ddl.py", line 72, in _execute_on_connection
    return connection._execute_ddl(self, multiparams, params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1041, in _execute_ddl
    ret = self._execute_context(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1287, in _execute_context
    self._handle_dbapi_exception(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1481, in _handle_dbapi_exception
    util.raise_(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1247, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedTable) relation "application" does not exist

[SQL: ALTER TABLE application ADD COLUMN created_at TIMESTAMP WITHOUT TIME ZONE]
(Background on this error at: http://sqlalche.me/e/f405)
Cleaning up... `

Docker status after this:
`sudo docker ps -a
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS                     PORTS               NAMES
2e73faaaaf72        dispatch-local      "dispatch database u…"   4 minutes ago       Exited (1) 4 minutes ago                       dispatch_web_run_4926996710db
dd61a2d4496d        dispatch-local      "dispatch --help"        4 minutes ago       Exited (0) 4 minutes ago                       dispatch_web_run_ccbde7802009
235a22ea542f        postgres:9.6        "docker-entrypoint.s…"   4 minutes ago       Exited (0) 4 minutes ago                       dispatch_postgres_1
`

`docker start 235a22ea542f`   # it works though and postgres container starts up with empty dispatcher database.

As per the error message, it is trying to run migration process at the time of installation only. **Please assist.**

**Comment by kevgliss** [comment]
Thanks for reporting, this should be fixed in #136

**Comment by B3lak0r** [comment]
> Thanks for reporting, this should be fixed in #136

Hi @kevgliss, I'm just having the same problem, could this be a random bug or it will persist without the appropriate fix?

**Comment by kevgliss** [comment]
@B3lak0r it's not random, #136 should have fixed this issue. Have you pulled in those changes (they are still currently only on the `develop` branch).

**Comment by B3lak0r** [comment]
> @B3lak0r it's not random, #136 should have fixed this issue. Have you pulled in those changes (they are still currently only on the `develop` branch).

No, I didn't, I was just trying the to deploy it using the docker's install script

**Comment by B3lak0r** [comment]
@kevgliss, any chances to get this solved or maybe a workaround?

**Comment by kevgliss** [comment]
@B3lak0r I haven't been able to re-create this after #136 was put in place. Could you provide any more details?

**Comment by B3lak0r** [comment]
> @B3lak0r I haven't been able to re-create this after #136 was put in place. Could you provide any more details?

Thanks in advance @kevgliss:  

Fresh and updated ubuntu 18.04 LTS VM:
 8 GB RAM
Docker version 19.03.8, build afacb8b7f0
docker-compose version 1.25.4, build 8d51620a
cloned dispatch-docker git
cd dispatch-docker
Edit docker-compose.yml adding lines to expose ports for web and dispatcher.
Filled my .env with the appropiatade data / credentials
Run ./install.sh

-------------------------------------------------------------------------------------------------------------

Successfully built fccce5e3f7df
Successfully tagged dispatch-local:latest

Docker images built.
Creating network "dispatch_default" with the default driver
Creating dispatch_postgres_1 ... done
Setting up database...
Starting dispatch_postgres_1 ... done                                                                                   Usage: dispatch [OPTIONS] COMMAND [ARGS]...

  Command-line interface to Dispatch.

Options:
  --version  Show the version and exit.
  --help     Show this message and exit.

Commands:
  contact    All commands for contact manipulation.
  database   Container for all dispatch database commands.
  incident   All commands for incident manipulation.
  plugins    All commands for plugin manipulation.
  scheduler  Container for all dispatch scheduler commands.
  server     Container for all dispatch server commands.
  term       All commands for term manipulation.
Starting dispatch_postgres_1 ... done                                                                                   INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade  -> e75e103693f2, Initial migration
INFO  [alembic.runtime.migration] Running upgrade e75e103693f2 -> d0501fc6be89, empty message
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1247, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
    cursor.execute(statement, parameters)
psycopg2.errors.UndefinedTable: relation "application" does not exist


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/bin/dispatch", line 8, in <module>
    sys.exit(entrypoint())
  File "/usr/local/lib/python3.8/site-packages/dispatch/cli.py", line 835, in entrypoint
    dispatch_cli()
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/dispatch/cli.py", line 612, in upgrade_database
    alembic_command.upgrade(alembic_cfg, revision, sql=sql, tag=tag)
  File "/usr/local/lib/python3.8/site-packages/alembic/command.py", line 298, in upgrade
    script.run_env()
  File "/usr/local/lib/python3.8/site-packages/alembic/script/base.py", line 489, in run_env
    util.load_python_file(self.dir, "env.py")
  File "/usr/local/lib/python3.8/site-packages/alembic/util/pyfiles.py", line 98, in load_python_file
    module = load_module_py(module_id, path)
  File "/usr/local/lib/python3.8/site-packages/alembic/util/compat.py", line 184, in load_module_py
    spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/local/lib/python3.8/site-packages/dispatch/alembic/env.py", line 84, in <module>
    run_migrations_online()
  File "/usr/local/lib/python3.8/site-packages/dispatch/alembic/env.py", line 78, in run_migrations_online
    context.run_migrations()
  File "<string>", line 8, in run_migrations
  File "/usr/local/lib/python3.8/site-packages/alembic/runtime/environment.py", line 846, in run_migrations
    self.get_context().run_migrations(**kw)
  File "/usr/local/lib/python3.8/site-packages/alembic/runtime/migration.py", line 520, in run_migrations
    step.migration_fn(**kw)
  File "/usr/local/lib/python3.8/site-packages/dispatch/alembic/versions/d0501fc6be89_.py", line 21, in upgrade
    op.add_column("application", sa.Column("created_at", sa.DateTime(), nullable=True))
  File "<string>", line 8, in add_column
  File "<string>", line 3, in add_column
  File "/usr/local/lib/python3.8/site-packages/alembic/operations/ops.py", line 1929, in add_column
    return operations.invoke(op)
  File "/usr/local/lib/python3.8/site-packages/alembic/operations/base.py", line 374, in invoke
    return fn(self, operation)
  File "/usr/local/lib/python3.8/site-packages/alembic/operations/toimpl.py", line 132, in add_column
    operations.impl.add_column(table_name, column, schema=schema, **kw)
  File "/usr/local/lib/python3.8/site-packages/alembic/ddl/impl.py", line 237, in add_column
    self._exec(base.AddColumn(table_name, column, schema=schema))
  File "/usr/local/lib/python3.8/site-packages/alembic/ddl/impl.py", line 140, in _exec
    return conn.execute(construct, *multiparams, **params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 984, in execute
    return meth(self, multiparams, params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/sql/ddl.py", line 72, in _execute_on_connection
    return connection._execute_ddl(self, multiparams, params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1041, in _execute_ddl
    ret = self._execute_context(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1287, in _execute_context
    self._handle_dbapi_exception(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1481, in _handle_dbapi_exception
    util.raise_(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1247, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedTable) relation "application" does not exist

[SQL: ALTER TABLE application ADD COLUMN created_at TIMESTAMP WITHOUT TIME ZONE]
(Background on this error at: http://sqlalche.me/e/f405)
Cleaning up...

**Comment by kevgliss** [comment]
What commit are you on?

**Comment by B3lak0r** [comment]
> What commit are you on?

@kevgliss I just recreated the issue with the latest docker-dispatch repo, having the same result:

Successfully built 48822d77cc50
Successfully tagged dispatch-local:latest

Docker images built.
Creating network "dispatch_default" with the default driver
Creating dispatch_postgres_1 ... done
Setting up database...
Starting dispatch_postgres_1 ... done                                                                                   Usage: dispatch [OPTIONS] COMMAND [ARGS]...

  Command-line interface to Dispatch.

Options:
  --version  Show the version and exit.
  --help     Show this message and exit.

Commands:
  contact    All commands for contact manipulation.
  database   Container for all dispatch database commands.
  incident   All commands for incident manipulation.
  plugins    All commands for plugin manipulation.
  scheduler  Container for all dispatch scheduler commands.
  server     Container for all dispatch server commands.
  term       All commands for term manipulation.
Starting dispatch_postgres_1 ... done                                                                                   INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade  -> e75e103693f2, Initial migration
INFO  [alembic.runtime.migration] Running upgrade e75e103693f2 -> d0501fc6be89, empty message
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1247, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
    cursor.execute(statement, parameters)
psycopg2.errors.UndefinedTable: relation "application" does not exist


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/bin/dispatch", line 8, in <module>
    sys.exit(entrypoint())
  File "/usr/local/lib/python3.8/site-packages/dispatch/cli.py", line 835, in entrypoint
    dispatch_cli()
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/dispatch/cli.py", line 612, in upgrade_database
    alembic_command.upgrade(alembic_cfg, revision, sql=sql, tag=tag)
  File "/usr/local/lib/python3.8/site-packages/alembic/command.py", line 298, in upgrade
    script.run_env()
  File "/usr/local/lib/python3.8/site-packages/alembic/script/base.py", line 489, in run_env
    util.load_python_file(self.dir, "env.py")
  File "/usr/local/lib/python3.8/site-packages/alembic/util/pyfiles.py", line 98, in load_python_file
    module = load_module_py(module_id, path)
  File "/usr/local/lib/python3.8/site-packages/alembic/util/compat.py", line 184, in load_module_py
    spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/local/lib/python3.8/site-packages/dispatch/alembic/env.py", line 84, in <module>
    run_migrations_online()
  File "/usr/local/lib/python3.8/site-packages/dispatch/alembic/env.py", line 78, in run_migrations_online
    context.run_migrations()
  File "<string>", line 8, in run_migrations
  File "/usr/local/lib/python3.8/site-packages/alembic/runtime/environment.py", line 846, in run_migrations
    self.get_context().run_migrations(**kw)
  File "/usr/local/lib/python3.8/site-packages/alembic/runtime/migration.py", line 520, in run_migrations
    step.migration_fn(**kw)
  File "/usr/local/lib/python3.8/site-packages/dispatch/alembic/versions/d0501fc6be89_.py", line 21, in upgrade
    op.add_column("application", sa.Column("created_at", sa.DateTime(), nullable=True))
  File "<string>", line 8, in add_column
  File "<string>", line 3, in add_column
  File "/usr/local/lib/python3.8/site-packages/alembic/operations/ops.py", line 1929, in add_column
    return operations.invoke(op)
  File "/usr/local/lib/python3.8/site-packages/alembic/operations/base.py", line 374, in invoke
    return fn(self, operation)
  File "/usr/local/lib/python3.8/site-packages/alembic/operations/toimpl.py", line 132, in add_column
    operations.impl.add_column(table_name, column, schema=schema, **kw)
  File "/usr/local/lib/python3.8/site-packages/alembic/ddl/impl.py", line 237, in add_column
    self._exec(base.AddColumn(table_name, column, schema=schema))
  File "/usr/local/lib/python3.8/site-packages/alembic/ddl/impl.py", line 140, in _exec
    return conn.execute(construct, *multiparams, **params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 984, in execute
    return meth(self, multiparams, params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/sql/ddl.py", line 72, in _execute_on_connection
    return connection._execute_ddl(self, multiparams, params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1041, in _execute_ddl
    ret = self._execute_context(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1287, in _execute_context
    self._handle_dbapi_exception(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1481, in _handle_dbapi_exception
    util.raise_(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1247, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedTable) relation "application" does not exist

[SQL: ALTER TABLE application ADD COLUMN created_at TIMESTAMP WITHOUT TIME ZONE]
(Background on this error at: http://sqlalche.me/e/f405)
Cleaning up...

**Comment by B3lak0r** [comment]
@kevgliss I know that you guys dont run dispatch on docker internally, so could it be possible to know how to do it that way?

**Comment by kevgliss** [comment]
Okay, so I think I know what _may_ be happening:

The initial issue was fixed, but you likely already had a database created in your postgres docker image. I would drop that imagine entirely (including data and re-running the script). 

I've also put together, #155 which removes the intermediate migrations for applications. We recently removed the applications table (in favor of tags), and here when trying to run an upgrade it can't find that deprecated table.

**Comment by crspeller** [comment]
@kevgliss Working though this as well. Using your #155 branch I run into this now:

```
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade  -> e75e103693f2, Initial migration
INFO  [alembic.runtime.migration] Running upgrade e75e103693f2 -> e3d9f5ca6958, Adding reported_at timestamp
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1247, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
    cursor.execute(statement, parameters)
psycopg2.errors.UndefinedTable: relation "incident" does not exist


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/bin/dispatch", line 8, in <module>
    sys.exit(entrypoint())
  File "/usr/local/lib/python3.8/site-packages/dispatch/cli.py", line 835, in entrypoint
    dispatch_cli()
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/dispatch/cli.py", line 612, in upgrade_database
    alembic_command.upgrade(alembic_cfg, revision, sql=sql, tag=tag)
  File "/usr/local/lib/python3.8/site-packages/alembic/command.py", line 298, in upgrade
    script.run_env()
  File "/usr/local/lib/python3.8/site-packages/alembic/script/base.py", line 489, in run_env
    util.load_python_file(self.dir, "env.py")
  File "/usr/local/lib/python3.8/site-packages/alembic/util/pyfiles.py", line 98, in load_python_file
    module = load_module_py(module_id, path)
  File "/usr/local/lib/python3.8/site-packages/alembic/util/compat.py", line 184, in load_module_py
    spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/local/lib/python3.8/site-packages/dispatch/alembic/env.py", line 84, in <module>
    run_migrations_online()
  File "/usr/local/lib/python3.8/site-packages/dispatch/alembic/env.py", line 78, in run_migrations_online
    context.run_migrations()
  File "<string>", line 8, in run_migrations
  File "/usr/local/lib/python3.8/site-packages/alembic/runtime/environment.py", line 846, in run_migrations
    self.get_context().run_migrations(**kw)
  File "/usr/local/lib/python3.8/site-packages/alembic/runtime/migration.py", line 520, in run_migrations
    step.migration_fn(**kw)
  File "/usr/local/lib/python3.8/site-packages/dispatch/alembic/versions/e3d9f5ca6958_.py", line 21, in upgrade
    op.add_column("incident", sa.Column("reported_at", sa.DateTime(), nullable=True))
  File "<string>", line 8, in add_column
  File "<string>", line 3, in add_column
  File "/usr/local/lib/python3.8/site-packages/alembic/operations/ops.py", line 1929, in add_column
    return operations.invoke(op)
  File "/usr/local/lib/python3.8/site-packages/alembic/operations/base.py", line 374, in invoke
    return fn(self, operation)
  File "/usr/local/lib/python3.8/site-packages/alembic/operations/toimpl.py", line 132, in add_column
    operations.impl.add_column(table_name, column, schema=schema, **kw)
  File "/usr/local/lib/python3.8/site-packages/alembic/ddl/impl.py", line 237, in add_column
    self._exec(base.AddColumn(table_name, column, schema=schema))
  File "/usr/local/lib/python3.8/site-packages/alembic/ddl/impl.py", line 140, in _exec
    return conn.execute(construct, *multiparams, **params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 984, in execute
    return meth(self, multiparams, params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/sql/ddl.py", line 72, in _execute_on_connection
    return connection._execute_ddl(self, multiparams, params)
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1041, in _execute_ddl
    ret = self._execute_context(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1287, in _execute_context
    self._handle_dbapi_exception(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1481, in _handle_dbapi_exception
    util.raise_(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 1247, in _execute_context
    self.dialect.do_execute(
  File "/usr/local/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedTable) relation "incident" does not exist

[SQL: ALTER TABLE incident ADD COLUMN reported_at TIMESTAMP WITHOUT TIME ZONE]
(Background on this error at: http://sqlalche.me/e/f405)
Cleaning up...
```

**Comment by kevgliss** [comment]
@crspeller do you have your database running? Can you login and show me the existing tables?

**Comment by crspeller** [comment]
@kevgliss Yup no tables:
```
dispatch=# \dt
Did not find any relations.
```

Is it possible the install script is just missing the `docker-compose run web database init` command? After running that I get:
```
dispatch=# \dt
                           List of relations
 Schema |                    Name                    | Type  |  Owner   
--------+--------------------------------------------+-------+----------
 public | alembic_version                            | table | dispatch
 public | assoc_incident_terms                       | table | dispatch
 public | assoc_individual_contact_incident_priority | table | dispatch
 public | assoc_individual_contact_incident_type     | table | dispatch
 public | assoc_individual_contact_terms             | table | dispatch
 public | conference                                 | table | dispatch
 public | conversation                               | table | dispatch
 public | definition                                 | table | dispatch
 public | definition_teams                           | table | dispatch
 public | definition_terms                           | table | dispatch
 public | document                                   | table | dispatch
 public | document_incident                          | table | dispatch
 public | document_incident_priority                 | table | dispatch
 public | document_incident_type                     | table | dispatch
 public | document_terms                             | table | dispatch
 public | group                                      | table | dispatch
 public | incident                                   | table | dispatch
 public | incident_priority                          | table | dispatch
 public | incident_type                              | table | dispatch
 public | individual_contact                         | table | dispatch
 public | participant                                | table | dispatch
 public | participant_role                           | table | dispatch
 public | policy                                     | table | dispatch
 public | recommendation                             | table | dispatch
 public | recommendation_accuracy                    | table | dispatch
 public | recommendation_documents                   | table | dispatch
 public | recommendation_incident_priorities         | table | dispatch
 public | recommendation_incident_types              | table | dispatch
 public | recommendation_individual_contacts         | table | dispatch
 public | recommendation_services                    | table | dispatch
 public | recommendation_team_contacts               | table | dispatch
 public | recommendation_terms                       | table | dispatch
 public | service                                    | table | dispatch
 public | service_incident                           | table | dispatch
 public | service_incident_priority                  | table | dispatch
 public | service_incident_type                      | table | dispatch
 public | service_terms                              | table | dispatch
 public | status_report                              | table | dispatch
 public | storage                                    | table | dispatch
 public | tag                                        | table | dispatch
 public | tags_incidents                             | table | dispatch
 public | task                                       | table | dispatch
 public | team_contact                               | table | dispatch
 public | team_contact_incident                      | table | dispatch
 public | team_contact_incident_priority             | table | dispatch
 public | team_contact_incident_type                 | table | dispatch
 public | team_contact_terms                         | table | dispatch
 public | term                                       | table | dispatch
 public | ticket                                     | table | dispatch
(49 rows)
```

**Comment by crspeller** [comment]
@kevgliss Can confirm the installation script finishes after running that command. Still can't get the UI accessible, but it looks like a separate issue. 

Is there a more appropriate place to ask for setup assistance or should I file another issue if it comes to that?

**Comment by kevgliss** [comment]
@crspeller that's interesting, I was hoping #136 would essentially do just that (create, everything, as needed) perhaps something is creating that database, but not the tables... will look.

For other issues, please feel free to open an issue here for help.

**Comment by B3lak0r** [comment]
Guys, finally got it working, check out the solution https://github.com/Netflix/dispatch-docker/issues/32#issuecomment-604447519
